challenge_storing_groceries/src/challenge_storing_groceries/near_object_designator.py

```diff
@@ -180,3 +176,23 @@ def _determine_points_of_interest(self, near_entity, radius, spacing):
     def __repr__(self):
         return "NearObjectSpotDesignator(place_location_designator={}, name='{}', area='{}')"\
                     .format(self.place_location_designator, self._name, self._area)
+
+
+if __name__ == '__main__':
+    import sys
+    from robot_skills.get_robot import get_robot
+    from robot_smach_states.util.designators import EntityByIdDesignator
+
+    if len(sys.argv) < 2:
+        print(f"usage: python {sys.argv[0]} [entity ID]")
+        sys.exit()
+
+    rospy.init_node('test_inspect_shelves')
+
+    robot = get_robot("hero")
+
+    shelfDes = EntityByIdDesignator(robot, uuid="closet")
+    entityDes = EntityByIdDesignator(robot, uuid=sys.argv[1])
+
+    nearObjectDes = NearObjectSpotDesignator(robot, entityDes, shelfDes)
+    nearObjectDes.resolve()
```

Comment:
double check names and usage string. Or remove here and fix in https://github.com/tue-robotics/tue_robocup/pull/1270